tests/query_workflow.go

```diff
@@ -181,8 +180,10 @@ func (s *ClientFunctionalSuite) TestQueryWorkflow_QueryWhileBackoff() {
 
 	for _, tc := range testCases {
 		s.T().Run(tc.testName, func(t *testing.T) {
+			stv := testvars.New(t)
+			r := require.New(t)
```

Comment:
What's the benefit of doing it this way? Saving a few characters or is there more?